swap/src/cli/command.rs

```diff
@@ -50,25 +148,53 @@ pub enum Command {
         #[structopt(long = "counterpart-peer-id", default_value = DEFAULT_ALICE_PEER_ID)]
         alice_peer_id: PeerId,
 
-        #[structopt(
-        long = "counterpart-addr",
-        default_value = DEFAULT_ALICE_MULTIADDR
+        #[structopt(long = "counterpart-addr", default_value = DEFAULT_ALICE_MULTIADDR
         )]
         alice_addr: Multiaddr,
+
+        #[structopt(
+            long = "config",
+            help = "Provide a custom path to the configuration file. The configuration file must be a toml file.",
+            parse(from_os_str)
+        )]
+        config: Option<PathBuf>,
+
+        #[structopt(long, help = "Activate debug logging.")]
+        debug: bool,
```

Comment:
Seems a bit excessive to duplicate these everywhere. Can we use [`flatten`](https://docs.rs/structopt/0.3.21/structopt/index.html#flattening) to reduce this duplication?

Reply:
Agree with @thomaseizinger  and can confirm it is possible, we have done this for the configuration and other parameters before :)

Reply:
done